src/spec.md

```diff
@@ -122,7 +126,9 @@ webxdc.sendToChat({
 });
 ```
 
-When you need to send an empty file make sure you set `message.file.base64` to an empty string (`""`), forgetting to set `base64` or setting it to `null` it is an error. This is also important for Messenger Implementations, because you need to check for `typeof message.file.base64 === "string"` instead of `!message.file.base64`, which would not allow empty files.
+> When you need to send an empty file make sure you set `message.file.base64` to an empty string (`""`), forgetting to set `base64` or setting it to `null` it is an error. This is also important for Messenger Implementations, because you need to check for `typeof message.file.base64 === "string"` instead of `!message.file.base64`, which would not allow empty files.
+
+> If you want to store text don't use `btoa()`, rather use `message.file.plainText` directly, because [`btoa()` has problems with some unicode/emojis](https://developer.mozilla.org/en-US/docs/Web/API/btoa#unicode_strings)
```

Comment:
```suggestion
> If you want to send text don't use `btoa()`, rather use `message.file.plainText` directly, because [`btoa()` has problems with some unicode/emojis](https://developer.mozilla.org/en-US/docs/Web/API/btoa#unicode_strings)
```